.github/workflows/tag.yml

```diff
@@ -43,17 +43,34 @@ jobs:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
           
-      - name: Build and push
+      - name: Build and push root images
         uses: docker/build-push-action@v2
         with:
           context: .
           platforms: linux/386,linux/amd64,linux/arm64,linux/arm/v6,linux/arm/v7
           push: true
           build-args: VERSION=${{ steps.vars.outputs.tag }}
+          target: root
+          tags: |
+            mondoolabs/mondoo:${{ steps.vars.outputs.tag }}-root
```

Comment:
I know that we discussed that the default container should be the non-root version (the one running under the `mondoo` user). I agree that this makes most sense as it will make sure that users start with a more restricted environment and they will have to consciously opt-in for a `root` user in case they need it. However, we should keep in mind that this change will break the Mondoo operator node scanning the moment we release a new version of the containers with these changes.

Everyone running an old version of the operator will end up using the non-root container for scanning their nodes, which would mean certain queries may fail due to permission issues